department-of-veterans-affairs/va.gov-cms
# VACMS-15515: Updates to Metatag 2.0.

## Description
Updates Metatag module to version 2.0. 

We want to update Metatag module to version 2.0 to make metatag information available in JSON:API. We need to ensure that doing so does not change any existing functionality, specifically that metatag information is unchanged via GraphQL.

Closes https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15515

## Testing done
- Ensure GraphQL metatag data is unchanged across content types. 
- Ensure metatag information is available via JSON:API 
- See [here](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15515#issuecomment-1795506616) for details.

## Screenshots
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/6863534/d449aadf-5a7e-4a40-b1bd-564007b4d5f2)


## QA steps
1. Navigate to [Metatag admin](https://pr16060-61qltxalbpk1zwditxa3olxffojpdyah.ci.cms.va.gov/admin/config/search/metatag)
   - [x] Validate that Metatag configuration looks correct for each type (Global, Front page, etc.)
2. Navigate to [GraphQL Explorer](https://pr16060-61qltxalbpk1zwditxa3olxffojpdyah.ci.cms.va.gov/graphql/explorer) and run a few queries described in [research findings](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15515#issuecomment-1795506616)
   - [x] Validate that metatag information is present and unchanged from existing Metatag version.
3. Visit JSON:API endpoints described in research findings
   - [x] Validate that metatag information is present

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
